plugins/modules/ec2_vol.py

```diff
@@ -92,6 +92,11 @@
       - tag:value pairs to add to the volume after creation.
     default: {}
     type: dict
+  purge_tags:
+    description: Whether to remove existing tags that aren't passed in the C(tags) parameter
+    default: false
+    type: bool
+    version_added: 1.4.0
```

Comment:
1.4 shipped last week so this will need to be 2.0 (I don't currently expect a 1.5)